images/win/scripts/SoftwareReport/SoftwareReport.Browsers.psm1

```diff
@@ -32,6 +32,19 @@ $webDrivers = @{
     }
 }
 
+function Build-BrowserSection {
+    return @(
+        ($(Get-BrowserVersion -Browser "chrome")),
+        ($(Get-SeleniumWebDriverVersion -Driver "chrome")),
+        ($(Get-BrowserVersion -Browser "edge")),
+        ($(Get-SeleniumWebDriverVersion -Driver "edge")),
+        ($(Get-BrowserVersion -Browser "firefox")),
+        ($(Get-SeleniumWebDriverVersion -Driver "firefox")),
+        ($(Get-SeleniumWebDriverVersion -Driver "iexplorer")),
+        ($(Get-SeleniumVersion))
```

Comment:
nit: I think external brackets are not needed here.
```pwsh
 $(Get-BrowserVersion -Browser "chrome"),
 $(Get-SeleniumWebDriverVersion -Driver "chrome"),
```
should work here.

Reply:
brackets removed